content/talks/_archive.rst

```diff
@@ -1,4 +1,4 @@
-public: yes
+public: no
```

Comment:
Is this file used anywhere? Or just in the repo for historical purposes?

Reply:
That file has some data that we haven't transferred over into talk pages. We're keeping it around, so we don't lose the data - until someone has a moment to do something with it.